README.md

```diff
@@ -186,7 +194,7 @@ The purpose of this project is to create a new library that builds on luasystems
 
 #### Project size
 
-*  Small (90 hours)
+*  Small (150 hours)
```

Comment:
Actually, we can not determine arbitrarily the amount of hours.. can just say if it's small (90h), medium (175h) or large (350h).. however, we can say here that it might be small or medium, and let it to be defined in the application depending on the contributor's skill level.. HTH

Reply:
thx for the heads up. I'll make it medium then, and make the strech-goal a regular one

Reply:
done